assets/js/googlesitekit/modules/create-module-store.js

```diff
@@ -127,6 +130,7 @@ export function createModuleStore( slug, args = {} ) {
 		);
 
 		const submitChangesStore = createSubmitChangesStore( {
+			isSettingsLoading,
```

Comment:
Let's not add this to any of the utility `create*Store`s for now. I see why you added it here but I think it would be better to combine it separately. We can create a mini store for this below as essentially `{ selectors: { isSettingsLoading } }`

This isn't needed/used at all within the submit changes store so let's not add it just to include here.